specs/eip4844/p2p-interface.md

```diff
@@ -129,18 +130,45 @@ Per `context = compute_fork_digest(fork_version, genesis_validators_root)`:
 
 **Protocol ID:** `/eth2/beacon_chain/req/beacon_blocks_by_root/2/`
 
-The EIP-4844 fork-digest is introduced to the `context` enum to specify EIP-4844 beacon block type.
+`BeaconBlocksByRoot v2` will be disabled in favor of `BeaconBlockAndBlobsSidecarByRoot v1` for tighter coupling of blocks and sidecars.
 
-Per `context = compute_fork_digest(fork_version, genesis_validators_root)`:
+Clients MUST disable `BeaconBlocksByRoot v2` after `EIP4844_FORK_EPOCH`.
```

Comment:
what if I want to get a beacon block by root from before the 4844 fork?

seems like we want to keep `BeaconBlocksByRootV2` for this purpose **or** support previous types with the context enum like in the `ByRange` spec here

Reply:
Makes sense. I added back the previous types except for 4844 type